src/main.c

```diff
@@ -158,6 +166,27 @@ int main(int argc, char **argv) {
                 altitude_db_init((AltitudeDB *)packet_pos, last_time, 1000 * strtod(strtok(NULL, ":"), NULL));
                 break;
 
+            case DTYPE_ACCELERATON:
+                // if (!room_for_block(sizeof(AccelerationDB))) {
+                //     continue;
+                // }
+                just_added_block_size = sizeof(AccelerationDB);
+                add_block_header(DATA_ACCEL, just_added_block_size);
+                acceleration_db_init((AccelerationDB *)packet_pos, last_time, strtod(strtok(NULL, ","), NULL) / 100,
+                                     strtod(strtok(NULL, ","), NULL) / 100, strtod(strtok(NULL, ","), NULL) / 100);
+                break;
+
+            case DTYPE_VELOCITY:
+                // if (!room_for_block(sizeof(AngularVelocityDB))) {
+                //     continue;
+                // }
+                just_added_block_size = sizeof(AngularVelocityDB);
+                add_block_header(DATA_ANGULAR_VEL, just_added_block_size);
+                angular_velocity_db_init((AngularVelocityDB *)packet_pos, last_time,
+                                         strtod(strtok(NULL, ","), NULL) / 10, strtod(strtok(NULL, ","), NULL) / 10,
```

Comment:
Same idea here, multiplication not division. Packet spec is possibly not well worded, if you have any suggestions on a better way to word the descriptions lmk (I'm assuming it was just a misunderstanding because of the conversation we had on Discord).